network/src/server.rs

```diff
@@ -72,37 +77,60 @@ impl Server {
             sender,
             sync_handler_lock,
             connection_frequency,
+            request_manager,
         }
     }
 
+    /// Returns the default bootnode addresses of the network.
+    pub fn get_bootnodes(&self) -> Vec<SocketAddr> {
+        // Initialize the vector to be returned.
+        let mut bootnode_addresses = Vec::with_capacity(self.context.bootnodes.len());
+        // Iterate through and parse the list of bootnode addresses.
+        for bootnode in self.context.bootnodes.iter() {
+            if let Ok(bootnode_address) = bootnode.parse::<SocketAddr>() {
```

Comment:
I'm assuming the address was already validated when it was added to `context.bootnodes`, but there's no harm in pattern-matching here :+1: